Hmm, getting 
```
 [ERROR] Failed to execute goal io.fabric8:docker-maven-plugin:0.31.0:start (docker-start) on project quarkus-reactive-pg-client-deployment: I/O Error: Unable to pull 'postgres:13.1' : unauthorized: authentication required -> [Help 1]
2021-02-03T18:07:37.0716816Z org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal io.fabric8:docker-maven-plugin:0.31.0:start (docker-start) on project quarkus-reactive-pg-client-deployment: I/O Error
```

--- a/docs/src/main/asciidoc/security-jwt.adoc
+++ b/docs/src/main/asciidoc/security-jwt.adoc
@@ -822,6 +822,7 @@ SmallRye JWT provides an API for securing the JWT claims using all of these opti
 ----
 
 Note you can use Smallrye JWT Build API without having to create MP JWT endpoints.
+It can also be excluded from `quarkus-smallrye-jwt` if MP JWT endpoints do not need to generate JWT tokens.
 
 === Create JwtClaimsBuilder and set the claims
 

--- a/extensions/smallrye-jwt/deployment/pom.xml
+++ b/extensions/smallrye-jwt/deployment/pom.xml
@@ -35,12 +35,12 @@
             <artifactId>quarkus-smallrye-jwt</artifactId>
         </dependency>
 
-        <!-- test dependencies -->
         <dependency>
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-smallrye-jwt-build-deployment</artifactId>
-            <scope>test</scope>
         </dependency>
+
+        <!-- test dependencies -->
         <dependency>
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-junit5-internal</artifactId>

--- a/extensions/smallrye-jwt/runtime/pom.xml
+++ b/extensions/smallrye-jwt/runtime/pom.xml
@@ -36,6 +36,10 @@
         </dependency>
         <dependency>
             <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-smallrye-jwt-build</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
             <artifactId>quarkus-jsonp</artifactId>
         </dependency>
     </dependencies>

